tests/OmniSharp.Roslyn.CSharp.Tests/CompletionFacts.cs

```diff
@@ -586,15 +575,10 @@ public MyClass2()
             Assert.Equal("text", item.TextEdit.NewText);
             Assert.All(completions.Items, c =>
             {
-                switch (c.Label)
-                {
-                    case "text:":
-                        Assert.True(c.Preselect);
-                        break;
-                    default:
-                        Assert.False(c.Preselect);
-                        break;
-                }
+                if (c.Label == "ToString")
```

Comment:
I honestly can't tell you why Roslyn wants to preselect this, but it does. The behavior in VSCode is that `ToString` is filtered out because of the prefix mismatch.